libs/backend/src/scan/cast_vote_records/build_cast_vote_record.ts

```diff
@@ -489,6 +512,7 @@ export function buildCastVoteRecord({
               ballotMarkingMode: 'machine',
             },
           }),
+          vxWriteIns: writeInCount > 0 ? writeInCount : undefined,
```

Comment:
Why does it need to be greater than 0 to be included? 0 seems like a valid count

Reply:
I'm sticking with the pattern I established in the other parts of this file of not including keys with undefined or empty values. Just want to be consistent. Can switch but would want to switch them all.

Reply:
Ok interesting I didn't catch that before. I think that general approach sounds fine. I would only recommend tweaking it for counts so we can communicate a message like: "yes, we did count this, but the count was 0" rather than "we don't know what the count was". I think in the other cases using `undefined` is clear.

Reply:
That's a good point, I've updated it in this PR.